apps/events/viewsets.py

```diff
@@ -366,7 +367,7 @@ class ReviewViewSet(viewsets.ModelViewSet):
     model = Review
     permission_classes = [IsAuthenticatedOrReadOnly, ReviewPermissions]
     lookup_url_kwarg = "review_id"
```

Comment:
Let' move all the endpoint to events/event_id on the urls.py level by using endpoint like:
```
router.register("events/(?P<event_id>[^/.]+)/ratings", RatingViewSet, basename="Events rating")
```

So in the end we will have events/event_id/reviews/review_id endpoints
And your response endpoint would become PATCH events/event_id/reviews/review_id/response